### PR TITLE
Updated linting

### PR DESCRIPTION
**Pull request by earobinson**
- ~~removed anonymous functions~~

### CODE REVIEW

**Comment by earobinson** [comment]
@azharkhan interested in your thoughts, trying to dry up some of our linting, and have no anonymous functions

File: spec/lib/jyson/jyson.undefinedValue.spec.js [hunk 1]
```diff
@@ -6,11 +6,12 @@ const expect = chai.expect;
...
 describe('jyson.undefinedValue.spec: basic undefined', () => {
-  beforeEach(() =>{
+  beforeEach(() => {
     this.templateFunction = jyson.buildTemplateFunction({
       a: 'a',
-      b: {b: 'b'},
-      c: function({object}) {
+      b: { b: 'b' },
+      // eslint-disable-next-line
+      c: function({ object }) {
```
Comment (earobinson) [inline spec/lib/jyson/jyson.undefinedValue.spec.js:14]: i wanted to keep one anonymous function for tests and what not.

File: spec/lib/jyson/jyson.nesting.spec.js [hunk 2]
```diff
@@ -5,7 +5,7 @@ const expect = chai.expect;
 const jyson = require('./../../../lib/jyson');
 
 describe('jyson.nesting.spec: a template with nesting', () => {
-  beforeEach(() =>{
+  beforeEach(() => {
```
Comment (earobinson) [inline spec/lib/jyson/jyson.nesting.spec.js:8]: this feels equally anonymous to me but passes the linter.

**Comment by azharkhan** [comment]
i do like the spacing and other changes, I just feel that if we are removing anonymous functions, especially since it doesn’t technically treats arrow functions as anon’s - then we can remove that part, and merge the rest.

**Review by azharkhan — COMMENTED** [review]
since `() => {}` function callbacks are pretty standard, i'm not sure if getting rid of them is a good idea. 

it will be quite exhausting to add naming to each `beforeEach` and callback functions.

but I'm open to suggestions.

**Review by azharkhan — APPROVED** [review]
as per our discussion on the chat

File: spec/lib/jyson/jyson.nesting.spec.js [hunk 3]
```diff
@@ -16,8 +16,8 @@ describe('jyson.nesting.spec: a template with nesting', () => {
   it('must convert an object to "json"', () => {
     const input = {
       a: 1,
-      b: {b: 2},
-      c: {c: {c:3}}
+      b: { b: 2 },
```
Comment (misshula) [inline spec/lib/jyson/jyson.nesting.spec.js:19]: ugh.. we've changed this back and forth a few times... 

is this just for jyson? or will our other repos have this too?

Comment (misshula) [reply spec/lib/jyson/jyson.nesting.spec.js:19]: NM. my bad. this is correct

**Review by misshula — APPROVED** [review]

File: spec/lib/jyson/jyson.nesting.spec.js (hunk 3 above)
Comment (earobinson) [reply spec/lib/jyson/jyson.nesting.spec.js:19]: all our open source code uses ed's style, all our closed source code uses hubba style. I plan on drying up all our open source code with the same rules.

**Pull request merged**